test/__snapshots__/warp-deprecated.spec.js.snap

```diff
@@ -0,0 +1,76 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`deprecated warp classes > Emits a warning if 's-text-link-hover' or 's-text-link-hover-active' are found 2`] = `
+[
+  "[93m[DEPRECATED][0m s-text-link-hover -> use 's-text-link'",
+  "[93m[DEPRECATED][0m s-text-link-hover-active -> use 's-text-link'",
```

Comment:
Did we remove hover and active states? So the link is the same in those states?

Reply:
Yes, those semantic color classes were removed and links have the same color in all states (see [taxonomy spreadsheet](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=888578831&range=B130))